docs/src/CommutativeAlgebra/affine_algebras.md

```diff
@@ -294,11 +294,38 @@ minimal_generating_set(I::MPolyQuoIdeal{<:MPolyDecRingElem})
 
 #### Intersection of Ideals
 
-```@docs
+```@julia
```

Comment:
is this change intended? why don't you want to include the docstring here?

Reply:
Please see here:
https://docs.oscar-system.org/dev/CommutativeAlgebra/affine_algebras/#Ideals-in-Affine-Algebras
When I include the docstring, a whole Rattenschwanz of methods for intersection will be printed. These happens occasionally and so far, none of our gurus had a solution for this problem. Please note: I duplicated the example using jldoctest in the .md file, so the example will run anyway.

Reply:
I think @thofma has some tool that might help in these cases. Maybe he could try it on this instance of the problem.

Reply:
There is no docstring for the signature
```
intersect(V::Vector{MPolyQuoIdeal{T}}) where T
```
so it just includes all of them. I think simply removing this should fix everything.

Reply:
and keep the `@docs`